### ISSUE TITLE
CTD 10-07-02 Ilteroi DLL

### ISSUE THREAD

**Issue by CokolwiekUnni** [LoneGazebo/Community-Patch-DLL#7257]
_1. Mod version (i.e Date - 12/1):_
10-07-2 with Ilteroi's dll https://github.com/LoneGazebo/Community-Patch-DLL/issues/7189

_2. Mod list (if using Vox Populi only, leave blank):_
https://steamcommunity.com/sharedfiles/filedetails/?id=177650706

_3. Error description:_ 
CTD during AI turn

_4. Steps to reproduce (optional):_
Press next turn. Happens always on the Zulus turn. I bought a new Civ5 Complete key and got new Steam account (I had only Civ there anyway) to check if fault is on my part due to some DLC missing, and I still get the same CTD. I sent the files to others and they were able to play it through. They sent the turn after to me, still crashes on AI turn. Happens frequently, wasn't able to finish a game through for some time now because of this.

Save game and CvMiniDump.dmp in the zip.

[Desktop.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/5640839/Desktop.zip)

![image](https://user-images.githubusercontent.com/75467145/101128719-82f61f80-3600-11eb-8ef9-43fd75c03d7d.png)

**Comment by ilteroi** [comment]
this is weird. it points to the same reason as #7249 but that was due to a change by Gazebo which isn't even in this build? don't understand it but that would mean it's fixed.

**Comment by CokolwiekUnni** [comment]
Thanks for quick reply. That would be strange indeed. It would also mean that my saves are dead anyway. I don't know if it was in the previous build, I assume no, since people only started to post on this issue after a new December version was released. For me CTD happens at different time on 10-07-02, I have a very stable games, and crashes appear only after hitting the information era.

This another game from information era. Can you check if it's the same issue?
[CvMiniDump (2).zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/5641206/CvMiniDump.2.zip)

**Comment by LoneGazebo** [comment]
@ilteroi wait what did I do?

**Comment by CokolwiekUnni** [comment]
Can it still be hovering issue if after sending to other they don't have the CTD with my saves and told me that in general they do not see any CTD in late eras?
Also I tried the thing from this thread https://forums.civfanatics.com/threads/stability-improvement.665199/
CORE_DISABLE_LUA_HOOKS and set to 1 but unfortunately it didn't help, sad duck.
I forgot to mention that I have some really small UI mods like top panel not condensed  and resources small icons but after disabling them the issue still persists.

**Comment by LoneGazebo** [comment]
I see now, for some reason my merge wasn't detecting your merge but I forced it and pushed the hotfix.

New cache makes more sense anyways. 💃

**Comment by RecursiveVision** [comment]
@CokolwiekUnni Open a new issue if you experience further crashes...
